ts/addTodo.ts

```diff
@@ -14,7 +15,7 @@ function addTodo(todo) {
             </button>
     `;
   todos.appendChild(newTodo);
-  todosDiv.appendChild(todos);
+  todosDiv?.appendChild(todos);
```

Comment:
Again lying to yourself. What do you expect from todosDiv? If it's always defined (I suppose so, meaning in can't be `undefined`) you should say it to typescript. 

If it can be undefined and this is expected, then handle this case.